### PR TITLE
feat: Provide stemmers as separate package.

### PR DESCRIPTION
**Pull request by ShogunPanda**
This was longly awaited.

### CODE REVIEW

File: packages/orama/package.json [hunk 1]
```diff
@@ -20,11 +20,6 @@
       "types": "./dist/components.d.ts",
       "import": "./dist/components.js",
       "require": "./dist/cjs/components.cjs"
-    },
-    "./stemmers/*": {
```
Comment (micheleriva) [inline packages/orama/package.json:24]: That means that we're still exporting ALL the stemmers together right? Can you please generate a single export per each stemmer?

File: packages/stemmers/package.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@orama/stemmers",
+  "version": "1.0.0-beta.15",
+  "type": "module",
+  "description": "Stemmers for Orama",
+  "sideEffects": false,
+  "main": "./dist/index.cjs",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
```
Comment (micheleriva) [inline packages/stemmers/package.json:10]: That means that we're exporting all the stemmers together, right? Can you please create an export for EACH stemmer here?

**Review by micheleriva — COMMENTED** [review]
We need to export individual stemmers. With the current implementation, we're gonna bundle all the stemmer together.

Also, documentation, README, and code examples need to reflect this change.

File: packages/stemmers/package.json (hunk 2 above)
Comment (ShogunPanda) [reply packages/stemmers/package.json:10]: We already have it: it's the * export on the next entry.
Are you saying we should not have an index at all?

**Comment by ShogunPanda** [comment]
Ok, will remove the index and update all code example accordingly.

**Comment by ShogunPanda** [comment]
@micheleriva I updated the code. Only individual stemmers can be imported now.
I update the documentation page about stemming. In the README I only found a reference to that page so nothing else to update.
Am I missing anything?

File: README.md [hunk 3]
```diff
@@ -101,26 +102,28 @@ await insert(db, {
...
-  }
+    rating: 4.5,
+  },
 })
 
 await insert(db, {
   name: 'Smart LED Bulb',
-  description: 'Control the lighting in your home with this energy-efficient smart LED bulb, compatible with most smart home systems.',
+  description:
+    'Control the lighting in your home with this energy-efficient smart LED bulb, compatible with most smart home systems.',
```
Comment (micheleriva) [inline README.md:112]: Pay attention to linting and prettier actions: here is putting the string on a new line (not needed). Do you happen to have a global prettier/eslint config?

File: package.json [hunk 4]
```diff
@@ -95,6 +95,7 @@
     "node": ">= 16.0.0"
   },
   "dependencies": {
-    "@changesets/cli": "^2.26.0"
+    "@changesets/cli": "^2.26.0",
+    "next": "^13.3.0"
```
Comment (micheleriva) [inline package.json:99]: Why is Next.js needed as a top-level dependency?

File: packages/orama/src/errors.ts [hunk 5]
```diff
@@ -7,6 +7,7 @@ const errors = {
   NO_LANGUAGE_WITH_CUSTOM_TOKENIZER: 'Do not pass the language option to create when using a custom tokenizer.',
   LANGUAGE_NOT_SUPPORTED: `Language "%s" is not supported.\nSupported languages are:\n - ${allLanguages}`,
   INVALID_STEMMER_FUNCTION_TYPE: `config.stemmer property must be a function.`,
+  MISSING_STEMMER: `As of version 1.0.0 @orama/orama does not ship non English stemmers by default. To solve this, please explicitly import and specify the "%s" stemmer from the package @orama/stemmers`,
```
Comment (micheleriva) [inline packages/orama/src/errors.ts:10]: I would also add a link to the documentation here

File: packages/orama/tests/diacritics.test.ts [hunk 6]
```diff
@@ -4,7 +4,7 @@ import { replaceDiacritics } from '../src/components/tokenizer/diacritics.js'
 t.test('Diacritics Replacer', t => {
   t.plan(1)
 
-  t.test('Should replace diacritics', t => {
+  t.test('should replace diacritics', t => {
```
Comment (micheleriva) [inline packages/orama/tests/diacritics.test.ts:7]: Is this a linter/prettier action?

**Review by micheleriva — COMMENTED** [review]
left some comments

File: README.md (hunk 3 above)
Comment (ShogunPanda) [reply README.md:112]: I think it was VS Code autolinting. Removed it!

File: package.json (hunk 4 above)
Comment (ShogunPanda) [reply package.json:99]: It was one of the attempt while Vercel deployments were failing. Removed as well.

File: packages/orama/tests/diacritics.test.ts (hunk 6 above)
Comment (ShogunPanda) [reply packages/orama/tests/diacritics.test.ts:7]: No: my OCD kicked in. Lol. Is that an issue?

File: packages/orama/src/errors.ts (hunk 5 above)
Comment (ShogunPanda) [reply packages/orama/src/errors.ts:10]: Good point. Done!

File: packages/orama/tests/diacritics.test.ts (hunk 6 above)
Comment (micheleriva) [reply packages/orama/tests/diacritics.test.ts:7]: Nope, just wondering

File: README.md [hunk 7]
```diff
@@ -101,26 +102,27 @@ await insert(db, {
...
-  }
+    rating: 4.3,
+  },
 })
 
 await insert(db, {
   name: 'Portable Charger',
-  description: 'Never run out of power on-the-go with this compact and fast-charging portable charger for your devices.',
+  description:
+    'Never run out of power on-the-go with this compact and fast-charging portable charger for your devices.',
```
Comment (micheleriva) [inline README.md:121]: This is still on a new line

Comment (ShogunPanda) [reply README.md:121]: It was a different line. Sob. Fixing it.

Comment (ShogunPanda) [reply README.md]: Done

**Pull request merged**